README.md

```diff
@@ -1,87 +1,77 @@
 
-Its primary focus is to serve the [`GLAM community`](https://en.wikipedia.org/wiki/GLAM_(industry_sector)) by providing a flexible, 
-respectful to identities and existing workflows, consistent and unified way of describing, storing, linking, exposing metadata and media assets. 
+Archipelago's primary focus is to serve the [`GLAM community`](https://en.wikipedia.org/wiki/GLAM_(industry_sector)) by providing a flexible, consistent, and unified way of describing, storing, linking, exposing metadata and media assets. We respect identities and existing workflows.
 
-All this under a bit different concept than the one we all have been used to in recent times. 
+All of this operates under a  different concept than the one we all have become used to in recent times. 
 
-We like to think this is not done by re-inventing the wheel, but by making sure the road is clean, leveled and with less obstacles
-than before, by removing some heavy weight from the top, some unneeded ballast, plus, of course, some well positioned innovations to make the ride enjoyable. 
+We like to think this is not done by re-inventing the wheel, but by making sure the road is clean, level, and with fewer obstacles
+than before. We do this by removing some heavy weight from the top, some unneeded ballast, plus, of course, some well positioned innovations to make the ride enjoyable. 
 
-We also like to say that Archipelago is like a Metadata Synthetizer (LFO anyone?) and we want to give you all the knobs, parameters, inputs and outputs to make the best out of it. 
-Still you can make "music" by just tapping the keyboard. 
+We also like to say that Archipelago is like a Metadata Synthetizer (LFO anyone?) and we want to give you all the knobs, parameters, inputs and outputs to make the best out of it. Still, you can make "music" by just tapping the keyboard. 
 
-Finally. It tries to stay humble, slim and nimble in nature with a small code base full of inline comments and `@todos`. 
+Finally, Archipelago tries to stay humble, slim, and nimble in nature with a small code base full of inline comments and `@todos`. 
 
-To get here we had to do a full stop first. Look around. Questioning everything we knew. Research and test (repeat) and then re-Architect slowly 
-on new and old assumptions, but specially new community values. 
+To get here we had to do a full stop first. Look around. Questioning everything we knew. Research and test (repeat) and then re-architect slowly on new and old assumptions, and especially new community values. 
 
 Learn [`what makes Archipelago so different`](docs/ourtake.md) and hopefully also a well known place.
 
 
-## Strawberryfields for ever
+## Strawberryfields Forever
```

Comment:
ha ha! This is a tricky one. So i went for the british version, the two words thing to distinguish it from the song (Even you all know the whole thing comes from there). So reading this http://www.grammarphobia.com/blog/2012/02/forever.html What do you think represents better SBF involvement?

Reply:
Very interesting! I'm learning something new. I'm all for the two-word version. How about "Strawberryfields For Ever" as the section header here? After all, I was seeking to standardize the casing in the section headers more than any other outcome.